README.md

```diff
@@ -20,6 +20,7 @@ The official repo is https://github.com/catarse/catarse
 To run this project you need to have:
 
 * Ruby 2.1.2
+* `ruby-devel`
```

Comment:
Actually the code runs on older versions as long as you adjust the Gemfile accordingly. I believe that this should be explicit in the README as well.